build/template.tex

```diff
@@ -79,14 +79,16 @@
     \phantomsection
     \conditionalLoT
 
-    % ABKÜRZUNGSVERZEICHNIS
-    % wird nur dann generiert, wenn mindestens ein mal \ac oder ein verwandter Befehl aufgerufen wurde und damit nicht leer ist
-    \ifnum\totvalue{acro_num}>0
-        \printacronyms
-        \addcontentsline{toc}{section}{Abkürzungsverzeichnis}
-        \cleardoublepage
+    % Hinweis: wenn sowohl Tabellenverzeichnis als auch Abbildungsverzeichnis leer sind, ist eine leere Seite vor dem Glossar
+    % in dem Falle bitte eine \cleardoublepage oben löschen
+
+    \if\CALLGLO 1 % alle Glossar-Einträge unbedingt erzeugen
+        \glsaddall
     \fi
 
+    \begingroup\let\newpage\relax\printnoidxglossaries\endgroup % Glossaries die leere Seite vor dem Glossar austreiben, wegen LoF, LoT

```

Comment:
Ich meine es kann das `\glsaddall` weggelassen werden. Dadurch werden immer nur verwendete Glossareinträge hinzugefügt, anstelle immer alle hinzuzufügen. Das Glossar wird so auch, bei keinem Aufruf, nicht generiert.
```suggestion
\printnoidxglossaries
```

Reply:
Das eine hat mit dem anderen nichts zu tun. Ich habe sowohl mit als auch ohne CALLGLO getestet und Glossaries hat konsistent immer einen Pagebreak erzeugt, auch und vor allem ohne glsaddall.

Besides, ich bin so einer, der die config auf 1 setzt, aus Angst, einen Glossar-Eintrag nicht zu referenzieren lol

Reply:
Also ich habe keine aktiven Glossarentries und nur folgenden Block aktiv in meiner `template.tex` stehen.
```latex
\conditionalLoT

\printnoidxglossary
\cleardoublepage

\setcounter{savepage}{\arabic{page}}

% MAIN CONTENT
\pagenumbering{arabic}
```
Es wird kein Glossar erzeugt und keine leere Seite entsteht. (Ich bitte dich das nochmal im aktuellen Zustand zu testen (bei mir wird nur eine extra Seite mit `\glsaddall` und ohne dem Hack erzeugt))

Und es sollte kein Glossar erzeugt werden, insofern etwas nicht referenziert wird meiner Meinung. (weil deswegen wird ja auch z.B. kein Literaturverzeichniss erzeugt wenn du keine Referenzen hast)

Zudem... sollte das Template nicht defaultmässig ein Glossar haben sondern ein Abkverzeichnis oder? Da DHGE Standard und so.

Reply:
Und zudem sollte im Kusche-Mode sollte das Glossar auch an einer [anderen Position](https://computerix.info/skripten/BaccArbeit.pdf#page=5) stehen.
> Das Glossar gehört hinter die Anlagen, aber vor das Literaturverzeichnis.